HIEUNTK24406H/baitapbuoi4/cau5

```diff
@@ -0,0 +1,11 @@
+#include<iostream>
+#include <algorithm>
+using namespace std;
+int main()
+{
+    int a , b , c , d;
+    cout << "nhap a , b , c , d : ";
+    cin >> a >> b >> c >> d;
+    cout << "vay so lon nhat la " << max(max(a,b) , max(c,d));
```

Comment:
a có rep cmt của e đó, xem thử cách đó nha
câu 4 anh đề xuất để code e ngắn hơn thôi chứ cũng ko hẳn phải làm lại

Reply:
> a có rep cmt của e đó, xem thử cách đó nha câu 4 anh đề xuất để code e ngắn hơn thôi chứ cũng ko hẳn phải làm lại

em thấy cách đó cũng ngắn nên cập nhật cho nhớ lun:)))

Reply:
> em thấy cách đó cũng ngắn nên cập nhật cho nhớ lun:)))
anh approve rồi đó, có nút **merge pull request** chưa em

Reply:
nó vẫn hiện merging is blocked a
> anh approve rồi đó, có nút **merge pull request** chưa em